modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java

```diff
@@ -443,4 +448,22 @@ public void addProperty(String key, Object value) {
     public Object getProperty(String key) {
         return propertyMap.get(key);
     }
+
+    /**
+     * Method to get the byte size of the message associated with this metadata
+     *
+     * @return content size of the message in bytes
+     */
+    public int getMessageContentSize() {
```

Comment:
I guess you can use [AndesMessageMetadata#getMessageContentLength](https://github.com/wso2/andes/blob/master/modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java#L395) for this. No need to implement another method.